package.json

```diff
@@ -36,10 +37,11 @@
     "react-chartjs-2": "^5.2.0",
     "react-dom": "18.2.0",
     "react-dropzone": "^14.2.3",
-    "react-redux": "^8.1.1",
+    "react-i18next": "^14.1.2",
```

Comment:
Where are these package requirements now coming from? i18next is still only used in cognition-ui.

Reply:
Got explanations in person: related to submodules shared across